src/controllers/itemDetails/index.js

```diff
@@ -1306,7 +1383,6 @@ function renderChildren(page, item) {
     } else if (item.Type == 'MusicArtist') {
         query.SortBy = 'PremiereDate,ProductionYear,SortName';
     }
-
```

Comment:
I would leave this empty line for readability.

Reply:
fixed on [f84ca77](https://github.com/jellyfin/jellyfin-web/pull/4733/commits/f84ca7711f789cb4065adef576e90f7117436792)